guix/Dockerfile

```diff
@@ -32,17 +31,17 @@ RUN wget -qO- "https://guix.carldong.io/signing-key.pub" | guix archive --author
 # Build Environment Setup
 # https://guix.gnu.org/manual/en/html_node/Build-Environment-Setup.html#Build-Environment-Setup
 
-RUN addgroup guixbuild
-RUN for i in $(seq -w 1 32);                    \
-  do                                            \
-    useradd -g guixbuild -G guixbuild           \
-            -d /var/empty -s "$(which nologin)" \
-            -c "Guix build user ${i}" --system  \
-          "guixbuilder${i}";                    \
-  done
-
-CMD guix-daemon --build-users-group=guixbuild   \
-                --substitute-urls="https://guix.carldong.io https://ci.guix.gnu.org"
+RUN groupadd --system guixbuild
+RUN for i in $(seq -w 1 10); do                  \
```

Comment:
Please leave this as `32`. I've increased it so I can perform guix builds with more than at most 10 builders.

Reply:
Addressed in 5ad2fb8ddcbc07df478126f513ef0f6d303d329c

Reply:
Hmm, not to spend too much time on this (and definitely don't prevent this from merge), but perhaps `$(nproc)` would be good here? I'm thinking number of builders should probably scale with number of threads?

Reply:
Heh, is there a reason to not set this to 256? :sweat_smile:

Reply:
@dongcarl Good idea, but the machine building the container can have different `$(nproc)` than the target machine using the container.

Reply:
> @dongcarl Good idea, but the machine building the container can have different `$(nproc)` than the target machine using the container.
Oh true... Let's just make it a build arg then with a default of 32?

> Heh, is there a reason to not set this to 256? 😅
I would think the scheduler might be very unhappy with us if we set this to 256 hehe

Reply:
Done in 8cb7890890e8a22f15b9cfb32a6cdf8e13dfa124.